INSTALL.md

```diff
@@ -12,8 +12,9 @@ Download using the [GitHub .zip download](https://github.com/dracula/terminal-ap
 
 #### Activating theme
 
-1.  _Terminal > Settings Tab_
-2.  Click _"Gear" icon_
-3.  Click _Import..._
-4.  Select the `Dracula.terminal` file
+1.  _Terminal > Preferences_
+2.  Click _Profiles tab_
+3.  In the left pane at the bottom, click _"...", then Import..._
```

Comment:
Profiles is correct, good catch.

But I still have a "Gear" Icon, not "…", to get to "Import." Perhaps we should just mention both?